src/status_im/keycard/unpair.cljs

```diff
@@ -124,31 +123,32 @@
-                                            (assoc-in [:keycard :pin]
-                                                      {:status      nil
-                                                       :error-label nil
-                                                       :on-verified nil}))
-      :keycard/persist-pairings         (dissoc pairings (keyword instance-uid))
-      :utils/show-popup                 {:title      (i18n/label (if keys-removed-from-card?
-                                                                   :t/profile-deleted-title
-                                                                   :t/database-reset-title))
-                                         :content    (i18n/label (if keys-removed-from-card?
-                                                                   :t/profile-deleted-keycard
-                                                                   :t/database-reset-content))
-                                         :on-dismiss #(re-frame/dispatch [:logout])}
-      ::key-storage/delete-multiaccount {:key-uid    key-uid
-                                         :on-success #(log/debug "[keycard] remove account ok")
-                                         :on-error   #(log/warn "[keycard] remove account: " %)}}
+     {:db                       (-> db
+                                    (update :profile/profiles-overview dissoc key-uid)
+                                    (assoc-in [:keycard :secrets] nil)
+                                    (update-in [:keycard :pairings]
+                                               dissoc
+                                               (keyword instance-uid))
+                                    (update-in [:keycard :pairings] dissoc instance-uid)
+                                    (assoc-in [:keycard :whisper-public-key] nil)
+                                    (assoc-in [:keycard :wallet-address] nil)
+                                    (assoc-in [:keycard :application-info] nil)
+                                    (assoc-in [:keycard :pin]
+                                              {:status      nil
+                                               :error-label nil
+                                               :on-verified nil}))
+      :keycard/persist-pairings (dissoc pairings (keyword instance-uid))
+      :utils/show-popup         {:title      (i18n/label (if keys-removed-from-card?
+                                                           :t/profile-deleted-title
+                                                           :t/database-reset-title))
+                                 :content    (i18n/label (if keys-removed-from-card?
+                                                           :t/profile-deleted-keycard
+                                                           :t/database-reset-content))
+                                 :on-dismiss #(re-frame/dispatch [:logout])}}
+     ;;should be reimplemented
```

Comment:
can you create a TODO with an issue tracked for this aswell/instead? 🙏

Reply:
there is not TODO and issue, we don't know when it will happen

Reply:
I'm a bit confused, for what reason can we not create a GH issue here?

Reply:
also it's better to just remove the commented out code 👍

Reply:
keycard is not supported in new design, when it will be supported we don't know, when it will be reimplemented someone who will be touching this code should be aware that this part need to be reimplemented as well